tests/manage/test_run_io_multiple_pods.py

```diff
@@ -1,19 +1,69 @@
 import pytest
 import logging
 import random
+from tests import helpers
 from ocs_ci.ocs import constants
 
 from ocs_ci.framework.testlib import ManageTest, tier2
-from tests.fixtures import (
-    create_rbd_storageclass, create_ceph_block_pool,
-    create_rbd_secret, create_pods, create_pvcs, create_project,
-    create_cephfs_secret, create_cephfs_storageclass
-)
 
 
 logger = logging.getLogger(__name__)
 
 
+@pytest.fixture()
+def create_pvcs(request):
+    """
+    Create multiple PVCs
+    """
+    class_instance = request.node.cls
+
+    def finalizer():
+        """
+        Delete multiple PVCs
+        """
+        if hasattr(class_instance, 'pvc_objs'):
```

Comment:
Instead of depending on a class attribute to store what pvcs were created we should instead just loop through the list of created pvs here and register a finalizer for each one. This fixture could then just return the list of created pvcs that we know will be properly torn down. If a test were to modify ``self.pvc_objs`` in anyway we would risk not tearing down pvcs.

Reply:
I agree, this fixture was part of `tests/fixtures.py` (from PR https://github.com/red-hat-storage/ocs-ci/pull/402/files#diff-8a66b2566ee3ea0285647f603db9174bR217) which I removed. It was used only by one test so I put to the file with the test and didn't change it much.